docs/_includes/variation-content.html

```diff
@@ -74,7 +74,7 @@ <h2 class="o-variation_group_title">
         {% endif %}
 
         {% if variation_group.variation_group_description %}
-        <div class="o-variation_group_description">
+        <div class="o-variation-group-description">
```

Comment:
Same as above, `o-variation-group` is the block, and description is the element`, so it would be `o-variation-group_description`.

Reply:
Updated!